build_tools/ci/run_matmul_test.sh

```diff
@@ -369,6 +375,7 @@ function run_matmul_test() {
   set +e
 
   compilation_flags="--iree-hal-target-backends=${target_backend} \
+                      --iree-amdaie-use-aie-backend=${aie_backend} \
```

Comment:
I think it shouldn't be called `aie-backend`.

Maybe we should adopt James' suggestion and use flags like

```
--iree-amdaie-tile-pipeline
--iree-amdaie-lower-to-aie-pipeline
```

Reply:
Done